india_banking_connector/host_to_host/doctype/h2h_status_log/h2h_status_log.py

```
@@ -0,0 +1,76 @@
+			if hasattr(host, "format_response"):
+				getattr(host, "format_response")(self.name, self.decrypted_data)
+
+	def set_formated_response(self) -> str:
+		host_doc = frappe.get_doc(self.host, self.host_name)
+
+		if host_doc.encrypt_payment_file:
+			try:
+				if hasattr(host_doc, "decrypt_file_content"):
+					decrypted_data = host_doc.decrypt_file_content(self.response)
+					self.decrypted_data = decrypted_data
+			except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)